standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1276,6 +1276,9 @@ public enum ConfVars {
         "hive.metastore.serdes.without.from.deserializer",
         "org.apache.iceberg.mr.hive.HiveIcebergSerDe",
         "SerDes which are providing the schema but do not need the 'from deserializer' comment for the columns."),
+    USE_TABLE_SERDES("metastore.use.table.serdes",
```

Comment:
not sure if we can turn on the feature by default, since this is a good performance improvement.